### ISSUE TITLE
No focus outline on the dropdown button

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-matching#194]
### Subject of the issue
Focus outline, when using keyboard, is explicitly disabled for each dropdown, before opening, at this line:

https://github.com/adaptlearning/adapt-contrib-matching/blob/46e9068c14a25ec7bce122f0793991b7f5bbb77e/less/dropdown.less#L4

**Comment by kirsty-hames** [comment]
I can't see why we would need to explicitly remove the focus outline here. This is part of the initial implementation where we [replaced select2 with native Adapt](https://github.com/adaptlearning/adapt-contrib-matching/pull/116). Historically there's been a lot of back and forth on how we display focus, e.g user agent vs theme style or variable. So this may have been whilst we were defining the outline style via the theme.

I'll check this across browsers/devices then raise a PR.
